pages/notifications.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+	<div class="dark:var(--main-bg) flex min-h-full flex-col gap-y-6 p-6">
+		<div class="flex flex-col items-center justify-between gap-y-2 sm:h-10 sm:flex-row">
+			<h1 class="text-2xl font-bold leading-8">Your notifications</h1>
+			<Button
+				v-if="displayedNotifications.length"
+				class="btn-mark-all-as-read"
+				@click="markAllNotificationsAsRead()"
+			>
+				<i class="pi pi-check-circle text-lg "/>
+				<span class="text-sm font-semibold">Mark all as read</span>
+			</Button>
+		</div>
+
+		<Accordion
+			v-if="displayedNotifications.length"
+			multiple
+			class="n-accordion"
+			:value="expandedIndexes"
+		>
+			<AccordionPanel
```

Comment:
On this page remove the collapsing functionality complete (probably don't even use that component).